nbproject/project.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this be ignored? we typically do not include IDE specific files. For example .idea (jetbrains) is ignored? Should these files be a available separately?

Perhaps we could have robo task to help with this.

Reply:
It is not IDE specific. I don't think the netbeans can edit only the SuiteCRM :) but it's okey I removed.